config/env_sample.json

```diff
@@ -85,19 +85,40 @@
     "/*  LTI Configuration": "*/",
     "/*  LTI is disabled by default": "*/",
     "STUDENT_DASHBOARD_LTI": false,
-    "/*  list LTI consumers as per https://github.com/rohitjose/django-lti-authusage make sure to change this secret!": "*/",
-    "PYLTI_CONFIG_CONSUMERS": { "LTI_CONSUMER_KEY_1": { "secret": "LTI_CONSUMER_SECRET_1" }},
-    "/*  LTI Default ID field.": "*/",
-    "/*  custom_canvas_use_login_id is the default, you could also use lis_person_sourcedid": "*/",
-    "LTI_PERSON_SOURCED_ID_FIELD": "custom_canvas_user_login_id",
-    "/*  LTI Email field.": "*/",
-    "LTI_EMAIL_FIELD": "lis_person_contact_email_primary",
-    "/*  LTI Course ID field.": "*/",
-    "LTI_CANVAS_COURSE_ID_FIELD": "custom_canvas_course_id",
-    "/*  LTI First Name.": "*/",
-    "LTI_FIRST_NAME": "lis_person_name_given",
-    "/*  LTI Last Name.": "*/",
-    "LTI_LAST_NAME": "lis_person_name_family",
+    "/*LTI 1.3 configuration":"*/",
+    "LTI_CONFIG": {
+        "/*canvas LMS domain nanme":"*/",
+        "https://canvas.test.instructure.com": [
```

Comment:
Depending on the point of view, that's wrong, too.

Do we want the actual hostname for UMich's Canvas or do we want to use fake names in this sample file?  If actual, `michedu` should be changed to `umich`.  Otherwise, it should be changed to something like `example` or `school_name_here`.

Reply:
I'd set all of these to be a generic and a comment that you need to fill this out for your institution. 

`https://canvas.instructure.com/`

Reply:
`https://example.canvas.instructure.com/`, maybe?